nixos/modules/services/networking/nat-iptables.nix

```diff
@@ -64,7 +64,7 @@ let
 
       # NAT the marked packets.
       ${optionalString (cfg.internalInterfaces != [ ]) ''
```

Comment:
Looks like the `internalInterfaces` option is not covered by the NixOS test, could you please add a test case with `internalInterfaces` option?

Reply:
Sorry, but wouldn't that be out of scope for this PR? I don't touch its interface, just existing internal implementation to improve it =/

Reply:
Without the test it's difficult to make sure that your changes aren't breaking anything and that they also actually make any difference